bot/cogs/memes_cog/memes_cog.py

```diff
@@ -120,6 +120,13 @@ async def crab(self, ctx, *, args='Bottom text\n is dead'):
         fnt_path = 'bot/cogs/memes_cog/assets/LemonMilk.otf'
         fnt = ImageFont.truetype(fnt_path, 11)
 
+        # Determine if there is a full rave or not
+        if str.isnumeric(args[0]):
```

Comment:
The flag should be a parameter in the crab function def
`    async def crab(self, ctx, is_rave: bool=True, *, args='Bottom text\n is dead'):` or something of this nature so these checks wont be needed